src/package-source-build/pack.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+
+# resolve $SOURCE until the file is no longer a symlink
+source="${BASH_SOURCE[0]}"
+scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
+while [[ -h $source ]]; do
+  scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
+  source="$(readlink "$source")"
+
+  # if $source was a relative symlink, we need to resolve it relative to the path where the
+  # symlink file was located
+  [[ $source != /* ]] && source="$scriptroot/$source"
+done
+
+scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
+
+"$scriptroot/../../eng/common/build.sh" --pack -c Release --warnaserror false $@ /p:Projects="$scriptroot/package.proj"
```

Comment:
Not sure if this is what you intended - this uses our bootstrap version of Arcade, not the version we build.  I'm fine with either since this is infra.

Reply:
Is this any different than other repos?  i.e. repos have their own version of arcade eng/common scripts with correspond to the arcade version that they use.  But, when built with source-build, they use the source-built version of arcade.